kratos/sources/kernel.cpp

```diff
@@ -18,11 +18,11 @@
 namespace Kratos {
 Kernel::Kernel() : mpKratosCoreApplication(Kratos::make_shared<KratosApplication>(
                 std::string("KratosMultiphysics"))) {
-    std::cout << " |  /           |             " << std::endl;
-    std::cout << " ' /   __| _` | __|  _ \\   __|" << std::endl;
-    std::cout << " . \\  |   (   | |   (   |\\__ \\ " << std::endl;
-    std::cout << "_|\\_\\_|  \\__,_|\\__|\\___/ ____/" << std::endl;
-    std::cout << "           Multi-Physics " << KRATOS_VERSION << std::endl;
+    KRATOS_INFO("Kratos Core Load") << " |  /           |             " << std::endl;
```

Comment:
Now in every line there will be an additional `Kratos Core Load`

Maybe you save the logo in a stringstream and pass it to the logger such that the `Kratos Core Load` appears only once

Furthermore now the logo wont show up any more if the verbosity is set to a higher level (StructuralMechanics does this in the tests)

Reply:
Thanks, I'll try a couple of things and push a proposal.